### DISCUSSION TITLE
Useful when turned off UAC on Win 11 22H2

### DISCUSSION THREAD

**Discussion by circcc** [HerMajestyDrMona/Windows11DragAndDropToTaskbarFix#93] [General]
When UAC is turned off in Win 11 22H2, the native feature to drag and drop files to the taskbar and the feature to move them by drag and drop to explorer tabs do not work. In the case of the former, this program is still essential and very useful.
Thank you very much.

**Comment by HerMajestyDrMona** [comment]
Thanks for the feedback.

What exactly do you mean by UAC turned off?
Do you mean this option?

![2023-03-06_17-22-03_User_Account_Control_Settings](https://user-images.githubusercontent.com/24290461/223169683-0862a91c-7bed-41eb-bb54-0601c08d3873.png)

or some other regedit / group policy changes?

**Comment by circcc** [comment]
[HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Policies\System]
"EnableLUA"=dword:00000000
